src/components/dialogs/parameters/shortcircuit/short-circuit-tension-table.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FunctionComponent, useCallback, useMemo } from 'react';
+import { Table, TableBody, TableCell, TableRow } from '@mui/material';
+import { useIntl } from 'react-intl';
+
+interface TensionTableProps {
+    isNominal: boolean;
+}
+const TensionTable: FunctionComponent<TensionTableProps> = ({ isNominal }) => {
```

Comment:
if TensionTable will be used only on shortcircuitparams, why isNominal is a props ? I think that you can make is a constant in this component.

Reply:
isNominal is based on InitialVoltageProfileMode selected value. For each InitialVoltageProfileMode value , we need to display a specific list of values. So isNominal should bee a props.